src/Interpolations.jl

```diff
@@ -4,6 +4,7 @@ export
     interpolate,
     interpolate!,
     extrapolate,
+    scale,
```

Comment:
In that case, I'd argue for `scale(itp, knots...)` as the common interface, returning a `ScaledInterpolant` in most cases, but where `scale(itp::GriddedInterpolation, knots...)` can special-case for re-use of the data array.